src/components/DelegateMultipleButton.jsx

```diff
@@ -187,6 +188,31 @@ const DelegateMultipleButton = props => {
     setLoadingDonations(false);
   };
 
+  const calcPledges = () => {
+    let sum = 0;
+    const sorted = [...delegations];
+    sorted.sort((a, b) => {
+      return b._amountRemaining.toNumber() - a._amountRemaining.toNumber();
+    });
+
+    if (sorted.length >= config.donationDelegateCountLimit) {
+      for (let i = 0; i < config.donationDelegateCountLimit; i += 1) {
+        sum += sorted[i]._amountRemaining.toNumber();
+      }
+    }
+    return sum;
+  };
+
+  const isLimitedDelegateCount = () => {
+    if (props.milestone && props.milestone.isCapped) {
```

Comment:
If milestone is not capped this alert maybe shown. Consider an uncapped milestone and a campaign which has donations of 12 different pledges

Reply:
If milestone is not capped and campaign has donations of 12 different pledges, the message should be shown.